xml/deploy_cephadm.xml

```diff
@@ -527,6 +527,23 @@ o- ssh ................................................... [Key Pair set]
   o- private_key ...... [53:b1:eb:65:d2:3a:ff:51:6c:e2:1b:ca:84:8e:0e:83]
   o- public_key ....... [53:b1:eb:65:d2:3a:ff:51:6c:e2:1b:ca:84:8e:0e:83]
 </screen>
+   <note>
+     <para>
+       The resulting exported JSON configuration includes a private SSH key.
```

Comment:
I am somewhat confused by how this paragraph begins. I could not find any mentions of `ceph-salt export` in this chapter before, so it feels strange to start with "the resulting exported JSON file" while not having talked about the option to export the configuration in the first place. I suggest to re-arrange this to explain that the config can be exported in JSON format using `ceph-salt export`, followed by an explanation why using `umask` is important here.

Reply:
So, confusingly, it's explained here: https://susedoc.github.io/doc-ses/master/single-html/ses-deployment/#deploy-cephadm-configure-export which is _after_ this section (configure export is 6.3.2.13 and SSH gen is 6.3.2.6)
Perhaps the configure export section should be moved above the SSH? Or, the umask note should be moved to the export chap... Thoughts?

Reply:
I think it's fine to conclude the configuration chapter with a note that the configuration can be exported for backup purposes, so the order of the section should stay. But I agree, I would move the umask note to the export part instead.

Reply:
Works for me.